src/ToDo.UI/App.xaml.host.cs

```diff
@@ -16,19 +16,15 @@ private static IHost BuildAppHost()
 #endif
 
 				// Add platform specific log providers
-				.UseLogging()
-
-				// Configure log levels for different categories of logging
-				.ConfigureLogging((context, logBuilder) =>
-				{
-					logBuilder
-							.SetMinimumLevel(
-								context.HostingEnvironment.IsDevelopment() ?
-									LogLevel.Trace :
-									LogLevel.Information)
-							.XamlLogLevel(LogLevel.Information)
-							.XamlLayoutLogLevel(LogLevel.Information);
-				})
+				.UseLogging(configure: (context, logBuilder) =>
+							// Configure log levels for different categories of logging
+							logBuilder
+									.SetMinimumLevel(
+										context.HostingEnvironment.IsDevelopment() ?
+											LogLevel.Trace :
```

Comment:
This is a bit too high, Trace in Uno logs a lot of things. Why not dev = Info, notdev = Warn?

Reply:
done